src/CitizenProject/CitizenProjectControllerSubscriber.php

```diff
@@ -17,8 +17,7 @@
  */
 class CitizenProjectControllerSubscriber implements EventSubscriberInterface
 {
-    const DEPARTMENT_CODES = ['13', '16', '19', '29', '30', '34', '38', '47', '54', '56', '73', '78', '79', '80', '81', '83', '84', '87', '89', '91', '92'];
-    const POSTAL_CODES = ['75005', '75006', '75017', '75018'];
```

Comment:
Are your sure those constants are not used?

Reply:
Yes